### DISCUSSION TITLE
Lithium + Amphetamines

### DISCUSSION THREAD

**Discussion by zelixir25** [TripSit/drugs#78] [Drug Interactions]
Rodent and human studies indicate that this combination is safe and that lithium may be neuroprotective when taken with amphetamines and that lithium can decrease the hyperactive effects.

Ago, Y., Tanaka, T., Kita, Y., Tokumoto, H., Takuma, K., & Matsuda, T. (2012). Lithium attenuates methamphetamine-induced hyperlocomotion and behavioral sensitization via modulation of prefrontal monoamine release. Neuropharmacology, 62(4), 1634–1639. https://doi.org/10.1016/j.neuropharm.2011.10.004

Gould, T. D., O’Donnell, K. C., Picchini, A. M., & Manji, H. K. (2007). Strain Differences in Lithium Attenuation of d-Amphetamine-Induced Hyperlocomotion: A Mouse Model for the Genetics of Clinical Response to Lithium. Neuropsychopharmacology, 32(6), 1321–1333. https://doi.org/10.1038/sj.npp.1301254

Zhou, Z., Wang, Y., Tan, H., Bharti, V., Che, Y., & Wang, J.-F. (2015). Chronic treatment with mood stabilizer lithium inhibits amphetamine-induced risk-taking manic-like behaviors. Neuroscience Letters, 603, 84–88. https://doi.org/10.1016/j.neulet.2015.07.027

**Comment by utaninja** [comment]
Looking at these studies, it seems to agree and support what you have said. However, I'm hesitant to push this to an issue as I'm personally not exactly sure what all of the studies are saying for this.

**Reply by zelixir25** [reply]
I can try to format my posts more as annotated bibliographies if that's helpful.

Ago, et al. (2012) studied the interaction of methamphetamine and lithium in mice, particularly its effects on behavioral sensitization and hyperactivity. Behavioral sensitization is the when repeated intermittent exposure to stimulants produces greater and longer-acting effects. It is thought to  They found that lithium reduced the hyperactivity expressed by mice who were also under the influence of methamphetamine. Gould, et al. (2007) found similar results for d-amphetamine specifically. 

Zhou, et al. (2015) used rats and found that rats who were given lithium and amphetamine engaged in fewer risk-taking behaviors than those who were treated with amphetamine alone.

I was finally able to track a human study today. Silverstone, et al. (1998) found that lithium did not attenuate the effects of d-amphetamine, which contradicts the rodent studies above. At the very least, based on seeing studies involving lithium and amphetamines with different hypotheses that are often unrelated to the goals of this project, I feel confident in saying that this combination is at least safe. I cannot make a judgement yet on whether it's synergistic, inhibitory, or neutral, though. I will keep looking for more studies that discuss subjective effects for this combo, especially since 1998 is quite a long time ago by now and the rodent studies are much more recent.

Additional reference:
Silverstone, P. H., Pukhovsky, A., & Rotzinger, S. (1998). Lithium does not attenuate the effects of d-amphetamine in healthy volunteers. Psychiatry Research, 79(3), 219–226. https://doi.org/10.1016/S0165-1781(98)00037-7

**Reply by utaninja** [reply]
Great thanks so much, aggregating this and pushing

**Comment by utaninja** [comment]
Lithium may be neuroprotective when taken with amphetamines and Lithium may decrease the hyperactive effects of amphetamines. Studies indicate this combination is safe.

Ago, Y., Tanaka, T., Kita, Y., Tokumoto, H., Takuma, K., & Matsuda, T. (2012). Lithium attenuates methamphetamine-induced hyperlocomotion and behavioral sensitization via modulation of prefrontal monoamine release. Neuropharmacology, 62(4), 1634–1639. https://doi.org/10.1016/j.neuropharm.2011.10.004

Gould, T. D., O’Donnell, K. C., Picchini, A. M., & Manji, H. K. (2007). Strain Differences in Lithium Attenuation of d-Amphetamine-Induced Hyperlocomotion: A Mouse Model for the Genetics of Clinical Response to Lithium. Neuropsychopharmacology, 32(6), 1321–1333. https://doi.org/10.1038/sj.npp.1301254

Zhou, Z., Wang, Y., Tan, H., Bharti, V., Che, Y., & Wang, J.-F. (2015). Chronic treatment with mood stabilizer lithium inhibits amphetamine-induced risk-taking manic-like behaviors. Neuroscience Letters, 603, 84–88. https://doi.org/10.1016/j.neulet.2015.07.027
Silverstone, P. H., Pukhovsky, A., & Rotzinger, S. (1998). Lithium does not attenuate the effects of d-amphetamine in healthy volunteers. Psychiatry Research, 79(3), 219–226. https://doi.org/10.1016/S0165-1781(98)00037-7

Silverstone, P. H., Pukhovsky, A., & Rotzinger, S. (1998). Lithium does not attenuate the effects of d-amphetamine in healthy volunteers. Psychiatry Research, 79(3), 219–226. https://doi.org/10.1016/S0165-1781(98)00037-7

**Comment by utaninja** [comment]
Pushed 🥳

**Reply by utaninja** [reply]
https://github.com/TripSit/drugs/pull/133
